extensions/exposition/features/io.feature

```diff
@@ -47,6 +49,7 @@ Feature: IO restrictions
     When the following request is received:
       """
       GET /pots/4c4759e6f9c74da989d64511df42d6f4/ HTTP/1.1
+      host: nex.toa.io
```

Comment:
There a huge amount of changes to add default `host` header in all features. A `host` header is required for HTTP/1 so wouldn't it better to add a default at htpp agent implementation?

Reply:
Again, I had the same idea.

My train of thought was that Agent's plain-text HTTP requests, in addition to their functional purposes, also serve educational ones. Requests without the `Host` header are technically invalid.

I'm still not sure about this decision. Since you've mentioned it and I have doubts, I'll implement it.